Gemfile.lock

```diff
@@ -1,30 +1,35 @@
 GEM
   remote: http://rubygems.org/
   specs:
-    addressable (2.3.8)
-    berkshelf (3.1.5)
-      addressable (~> 2.3.4)
-      berkshelf-api-client (~> 1.2)
+    addressable (2.5.2)
+      public_suffix (>= 2.0.2, < 4.0)
+    berkshelf (4.3.5)
+      addressable (~> 2.3, >= 2.3.4)
+      berkshelf-api-client (~> 2.0, >= 2.0.2)
       buff-config (~> 1.0)
       buff-extensions (~> 1.0)
       buff-shell_out (~> 0.1)
-      celluloid (~> 0.16.0.pre)
-      celluloid-io (~> 0.16.0.pre)
-      faraday (~> 0.9.0)
-      minitar (~> 0.5.4)
-      octokit (~> 3.0)
-      retryable (~> 1.3.3)
-      ridley (~> 4.0)
-      solve (~> 1.1)
-      thor (~> 0.18)
-    berkshelf-api-client (1.3.1)
+      celluloid (= 0.16.0)
+      celluloid-io (~> 0.16.1)
+      cleanroom (~> 1.0)
+      faraday (~> 0.9)
+      httpclient (~> 2.7)
+      minitar (~> 0.5, >= 0.5.4)
```

Comment:
This version specification installs minitar 0.6.1 and it's what we want, but I think it is not what the specification wanted 🤔